packages/frontend/src/welcome/welcome-screen.tsx

```diff
@@ -1,12 +1,99 @@
 import * as React from 'react'
 
-import {Link} from 'react-router-dom'
-
-export const Welcome = (): JSX.Element => (
-  <div>
-    <h1>Welcome</h1>
-    <Link to="/record">Record a message</Link>
-    <Link to="/messages">Messages</Link>
-    <Link to="/call">Call</Link>
-  </div>
+import Button from '@material-ui/core/Button'
+import Card from '@material-ui/core/Card'
+import CardActions from '@material-ui/core/CardActions'
+import CardContent from '@material-ui/core/CardContent'
+import Container from '@material-ui/core/Container'
+import Paper from '@material-ui/core/Paper'
+import Typography from '@material-ui/core/Typography'
+import {makeStyles, Theme, createStyles} from '@material-ui/core/styles'
+import {Link as RouterLink, Link} from 'react-router-dom'
+
+import {useUserSettings} from '../settings/use-user-settings'
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    mainContainer: {
+      maxWidth: 750,
+      margin: 'auto',
+    },
+    cardContainer: {
+      marginTop: theme.spacing(4),
+      display: 'flex',
+      flexDirection: 'row',
+      justifyContent: 'space-between',
+    },
+    settingsContainer: {
+      marginTop: theme.spacing(4),
+      marginLeft: theme.spacing(3),
+      marginRight: theme.spacing(3),
+      padding: theme.spacing(2),
+      backgroundColor: 'hsl(0, 100%, 93%)',
+    },
+    textAlign: {
+      textAlign: 'center',
+    },
+    card: {
+      flex: 1,
+      maxWidth: 300,
+    },
+  }),
 )
+
+const SettingsPrompt = () => {
+  const classes = useStyles()
+  return (
+    <Paper elevation={0} className={classes.settingsContainer}>
+      <Typography variant="h5" className={classes.textAlign}>
+        Configuration Missing!
+      </Typography>
+      <Typography variant="body1" style={{marginTop: 20}}>
+        It looks like this might be your first time using Introvert Activism. You will need to{' '}
+        <Link to="/settings">configure a few settings</Link> before jumping onto calls.
+      </Typography>
+    </Paper>
+  )
+}
+
+export const Welcome = (): JSX.Element => {
+  const classes = useStyles()
+  const [settings] = useUserSettings()
+  const settingsNotConfigured = !settings.accessToken || !settings.remoteApiOrigin
+
+  return (
+    <div className={classes.mainContainer}>
+      {settingsNotConfigured && (
+        <div>
+          <SettingsPrompt />
```

Comment:
Great addition! WDYT about expanding this and the warning in /call to flag if you haven't recorded a message yet to push you over there?

Reply:
That sounds good too!

Would that warning be in _welcome_ or in `/call`?

Reply:
I imagine in /welcome we'd just disable the link to /call and in `/call` we'd show the warning with a link to record a new message

Reply:
I'm confused. We'd have to enable the link, how else would you get to `/call`? No sense making someone clip the nav bar. 

So in `/welcome` we only require the settings setup, then we enable call, then in `/call` we'd have the same type of warning for having no calls setup.

Reply:
> We'd have to enable the link, how else would you get to /call? No sense making someone clip the nav bar.

It'd become enabled once messages have been recorded. The idea being we don't encourage them to go to call when there's nothing for them to do there (in fact I think it'd make sense to serve as an onboarding that doesn't show the link to call at all)

> So in /welcome we only require the settings setup, then we enable call, then in /call we'd have the same type of warning for having no calls setup.

That's fine too I supose